src/backend/api/Fusion.Resources.Api/Controllers/Requests/AnalyticsController.cs

```diff
@@ -85,6 +85,8 @@ private ApiPersonAbsenceForAnalytics(QueryPersonAbsenceBasic absence)
                 Comment = absence.Comment;
                 TaskDetails = (absence.TaskDetails != null) ? new ApiTaskDetails(absence.TaskDetails) : null;
 
+                Person = new ApiPerson(absence.Person!);
```

Comment:
Maybe rather do a nullcheck and throw your own error, instead of this potentially resulting in a nullreferror.. 
Or just do a check, if not null, set the property.. 

Kind of depends what we want as an outcome.. Do we want 1 row failing, or the whole dataset failing, if one row is invalid... ?

Reply:
All rows will have a person. But null check will be added.